src/Dapr.Workflow/Dapr.Workflow.csproj

```diff
@@ -2,7 +2,7 @@
 
   <!-- NuGet configuration -->
   <PropertyGroup>
-    <TargetFramework>net6</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5;net6</TargetFrameworks>
```

Comment:
Yes, it targets `netstandard2.0`, so we should be good with any of these targets for the SDK.

I would recommend `net6` instead of `net7`, however, since some users are hesitant to move to a non-LTS version of .NET.